packages/flutter/lib/src/rendering/object.dart

```diff
@@ -436,6 +436,15 @@ class PaintingContext {
     pushLayer(new OpacityLayer(alpha: alpha), painter, offset);
   }
 
+  /// Pushes an annotated region into the layer tree.
+  ///
+  /// See also:
+  ///
+  ///   * [Layer.findRegion], for an example of how this is used.
+  void pushAnnotatedRegion<T>(Offset offset, T value, PaintingContextCallback painter) {
```

Comment:
just use pushLayer directly at the callsite. We're trying to reduce the PaintingContext API to not have these methods; they just cost cycles and don't add any clarity.